Mathlib/Analysis/SpecialFunctions/ImproperIntegrals.lean

```diff
@@ -177,3 +177,32 @@ theorem integral_Ioi_cpow_of_lt {a : ℂ} (ha : a.re < -1) {c : ℝ} (hc : 0 < c
   simp_rw [neg_neg, Complex.norm_eq_abs, Complex.abs_cpow_eq_rpow_re_of_pos hx, Complex.add_re,
     Complex.one_re]
 #align integral_Ioi_cpow_of_lt integral_Ioi_cpow_of_lt
+
+private theorem intervalIntegral_one_div_one_add_sq_tendsto :
+    Tendsto (fun i => ∫ (x : ℝ) in -i..i, 1 / (1 + x ^ 2)) atTop (𝓝 π) := by
+  have := tendsto_nhds_of_tendsto_nhdsWithin arctan_atTop
+  convert Tendsto.add this this <;> simp
+
+theorem integrable_one_div_one_add_sq : Integrable fun (x : ℝ) ↦ 1 / (1 + x ^ 2) := by
+  have (x : ℝ) : ‖1 / (1 + x ^ 2)‖ = 1 / (1 + x ^ 2) := norm_of_nonneg (by positivity)
+  refine integrable_of_intervalIntegral_norm_tendsto π (fun i ↦ ?_) tendsto_neg_atTop_atBot
+    tendsto_id (by simpa only [this] using intervalIntegral_one_div_one_add_sq_tendsto)
+  by_cases hi : i = 0
+  · rewrite [hi, Set.Ioc_eq_empty (by norm_num)]; exact integrableOn_empty
+  · refine (intervalIntegral.intervalIntegrable_of_integral_ne_zero ?_).1
+    simp [← two_mul, arctan_ne_zero hi]
+
+theorem integral_Iic_one_div_one_add_sq {i : ℝ} :
+    ∫ (x : ℝ) in Set.Iic i, 1 / (1 + x ^ 2) = arctan i + (π / 2) :=
```

Comment:
I wonder if we have a stated preference for closed or open intervals in integrals; if we do, we should follow it.

Reply:
I extrapolated from the very small sample size of this file. Now might be the time to establish such a convention

Reply:
I would suggest closed (as already implemented here) since [intervalIntegral](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Integral/IntervalIntegral.html#intervalIntegral) uses `Set.Ioc` under the hood. That way if you were to subtract two of these `Set.Iic` integrals, you would get an `intervalIntegral` more easily.